libraries/entities/src/EntityItem.h

```diff
@@ -574,7 +576,7 @@ class EntityItem : public QObject, public SpatiallyNestable, public ReadWriteLoc
     void setSimulated(bool simulated) { _simulated = simulated; }
 
     const QByteArray getDynamicDataInternal() const;
-    bool stillHasGrabActions() const;
```

Comment:
The method above was renamed and moved public.

The method below is new and so far is only used internally so it can be private.